examples/using-reach-skip-nav/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
```

Comment:
I think you may be able to omit this fixture, or does it get created automatically?

Reply:
it gets created automatically